### ISSUE TITLE
OrderProcessingStarted is never triggered

### ISSUE THREAD

**Issue by csavelief** [vanilophp/framework#167]
Hi!

I cloned the [demo](https://github.com/vanilophp/demo) project and tried to catch the `OrderProcessingStarted` event.

I modified the `EventServiceProvider::$listen` variable to reference my listener :
```
    protected $listen = [
        'App\Events\Event' => [
            'App\Listeners\EventListener',
        ],
        OrderProcessingStarted::class => [
            SendOrderProcessingStartedNotification::class,
        ],
        OrderWasCancelled::class => [
            SendOrderCancelledNotification::class,
        ],
    ];
``` 

Moving in the `Admin > Orders` panel an order from :
- `pending` to `cancelled` triggers the `OrderWasCancelled` event ;
- `pending` to `processing` does not triggers the `OrderProcessingStarted` event.

Best,

Cyrille

**Comment by fulopattila122** [comment]
Thank you for the report. The `OrderProcessingStarted` event was added in Vanilo 3.7, but the Admin package hasn't been updated to all its features. It will be completed within a couple of days

**Comment by fulopattila122** [comment]
Please upgrade the vanilo/admin package with composer to v3.7 and the event will be triggered

**Comment by csavelief** [comment]
You are right : the issue is fixed on version 3.7

Thank you!
